Sources/Workspace/InitPackage.swift

```diff
@@ -316,10 +316,10 @@ public final class InitPackage {
         case .executable:
             content = """
                 @main
-                public struct \(typeName) {
-                    public private(set) var text = "Hello, World!"
+                struct \(typeName) {
+                    private(set) var text = "Hello, World!"
```

Comment:
I think we can also remove the `private(set)`? It was important in conjunction with `public` but seems fine for this to just be `internal` now.

Reply:
What do you think of just plainly embedding `"Hello, World!"` as a literal in that `print` invocation or making the property `static`? I tend to agree with some of the feedback I saw on this sample code that says that creating a one-off value of the type with `\(typeName)().text` feels weird, even `Self.text` feels somewhat nicer if it were `static`.

Reply:
I think we need the property to be able to write a test for it?

Reply:
Also cc @bitjammer who may have opinions on this

Reply:
I don't see any harm in removing the public modifiers for now, although this content could significantly change soon.

Reply:
Ok, thanks for the feedback. Merging as is, with just `public` removed for now.